buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -168,13 +167,13 @@ public void execute(Task task) {
     }
 
 
-    private void createListClustersTask(Project project, NamedDomainObjectContainer<ElasticsearchNode> container) {
+    private void createListClustersTask(Project project, NamedDomainObjectContainer<ElasticsearchCluster> container) {
         Task listTask = project.getTasks().create(LIST_TASK_NAME);
         listTask.setGroup("ES cluster formation");
         listTask.setDescription("Lists all ES clusters configured for this project");
         listTask.doLast((Task task) ->
             container.forEach(cluster ->
-                logger.lifecycle("   * {}: {}", cluster.getName(), cluster.getDistribution())
+                logger.lifecycle("   * {}: {}", cluster.getName(), cluster.getNumberOfNodes())
```

Comment:
Do we not care about the distribution anymore?

Reply:
Different nodes could be running a different one. 
This is just a task to list clusters that's not really used much so I figured this works for now. To make this really useful we would probably need to make it more like a report where you can see all details of the cluster configuration.